### DISCUSSION TITLE
v2ray-core v5使用有问题

### DISCUSSION THREAD

**Discussion by VoidRegion** [v2fly/v2ray-core#1953] [Q&A] [answered]
v2ray-core v5无法使用，听说是因为刚出还不完善，也不太想多折腾，请问有没有降级回v4内核的办法/教程。谢谢

**Comment by wenge110110** [comment, marked as answer]
处理方式1：
rm -rf /etc/systemd/system/v2ray.service.d/10-donot_touch_single_conf.conf
systemctl daemon-reload
systemctl restart v2ray
处理方式2：
vi /etc/systemd/system/v2ray.service.d/10-donot_touch_single_conf.conf
把ExecStart=/usr/local/bin/v2ray -config /usr/local/etc/v2ray/config.json
改为ExecStart=/usr/local/bin/v2ray run -config /usr/local/etc/v2ray/config.json
systemctl daemon-reload
systemctl restart v2ray
处理方式3：
切回老版本：
bash <(curl -L https://raw.githubusercontent.com/v2fly/fhs-install-v2ray/master/install-release.sh) --version v4.45.2 就好啦

**Comment by kulongwangzhi85** [comment]
./v2ray help能解决问题。
